x-pack/plugins/canvas/public/services/platform.ts

```diff
@@ -45,6 +45,7 @@ export const platformServiceFactory: CanvasServiceFactory<PlatformService> = (
     getUISetting: coreStart.uiSettings.get.bind(coreStart.uiSettings),
     setBreadcrumbs: coreStart.chrome.setBreadcrumbs,
     setRecentlyAccessed: coreStart.chrome.recentlyAccessed.add,
+    setFullscreen: coreStart.chrome.setIsVisible,
```

Comment:
I think you'll need to add an entry for setFullscreen in the type definition above and also stub it out in the `services/stubs/platform` file but other than that 👍